embrace-android-otel/src/main/kotlin/io/embrace/android/embracesdk/internal/otel/config/OtelSdkConfig.kt

```diff
@@ -24,6 +24,11 @@ import io.opentelemetry.semconv.incubating.OsIncubatingAttributes
 import io.opentelemetry.semconv.incubating.TelemetryIncubatingAttributes
 import java.util.concurrent.ConcurrentHashMap
 
+/**
+ * Globally referenceable flag that determines which OTel SDK implementation is in use for the production SDK at runtime.
+ */
+const val USE_KOTLIN_SDK: Boolean = false
```

Comment:
Do we have plans to make this part of RemoteConfig in a future PR, or will an end-user always have to set this value locally?